packages/settings-library/src/settings_library/efs.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+
+from pydantic import Field
+
+from .base import BaseCustomSettings
+
+
+class AwsEfsSettings(BaseCustomSettings):
+    EFS_DNS_NAME: str = Field(
```

Comment:
You could also have pydantic validate that the vlaue of the env-var is a valid DNS name. DNS names have very specific constraints beyond URLs or strings

Reply:
Yes I can add some pydantic custom validation for that :+1: